model/authorizations/v1/self_terms_review_resource.model

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright (c) 2020Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Manages terms review.
+resource SelfTermsReview {
+	// Reviews a user's status of Terms.
```

Comment:
I think many sentences here will be clearer if you say "terms acceptance" or "acceptance of terms" (or even "acceptance of Terms of Use") instead of just "terms"...

Reply:
// Check whether a user needs to acknowledge the terms for using OpenShift Dedicated and MOA

note: the term is called "Appendix 4 terms/conditions" from the epic, but I feel it's more clear just to say its intention.